### PR TITLE
feat(ifo): New IFO format

### PR DESCRIPTION
**Pull request by chef-jojo**
New IFO format 🤫

### ifo
Separate IFO current card and foldable card

## pools
refactor auto cake vault into VaultPool, using key 'cakeVault' or 'ifoPool'

There are lots of code in pool and ifo importing each other, should refactor them when we have time

### CODE REVIEW

File: src/config/constants/contracts.ts [hunk 1]
```diff
@@ -55,6 +55,10 @@ export default {
     56: '0xa80240Eb5d7E05d3F250cF000eEc0891d00b51CC',
     97: '',
   },
+  ifoPool: {
+    56: '0x0000',
```
Comment (chef-jojo) [inline src/config/constants/contracts.ts:59]: update address before merge

File: src/config/constants/pools.ts [hunk 2]
```diff
@@ -1,8 +1,22 @@
...
+export const vaultPoolConfig = {
+  [VaultKey.CakeVault]: {
+    name: 'Auto CAKE',
+    description: 'Automatic restaking',
+    autoCompoundFrequency: 5000,
+  },
+  [VaultKey.IfoPool]: {
+    name: 'IFO CAKE',
+    description: 'Stake CAKE to participate in IFO',
+    autoCompoundFrequency: 5000, // TODO: update IFO compound frequency
```
Comment (chef-jojo) [inline src/config/constants/pools.ts:16]: @Chef-Cheems What's the `autoCompoundFrequency` for this pool?

Comment (Chef-Cheems) [reply src/config/constants/pools.ts:16]: Lets say 2000.
It is not exact, just approximate. Depends how many times users stake or harvest.
To be honest the difference in shown APY would be tiny anyway between 2000 and 5000. The law of diminishing returns is kinda tough on the compounding, after certain compounding cycles it barely makes a difference.

File: src/views/Pools/components/PoolCard/StyledCard.tsx [hunk 3]
```diff
@@ -2,8 +2,10 @@ import styled from 'styled-components'
 import { Card } from '@pancakeswap/uikit'
 
 export const StyledCard = styled(Card)<{ isFinished?: boolean }>`
-  max-width: 352px;
+  max-width: 400px;
```
Comment (chef-jojo) [inline src/views/Pools/components/PoolCard/StyledCard.tsx:5]: fix this should be pass from upper component

Comment (chef-jojo) [reply src/views/Pools/components/PoolCard/StyledCard.tsx:5]: opps design changed back to 352

File: src/views/Pools/components/PoolsTable/Cells/TotalStakedCell.tsx [hunk 4]
```diff
@@ -21,6 +22,10 @@ const TotalStakedCell: React.FC<TotalStakedCellProps> = ({ pool }) => {
   const { t } = useTranslation()
   const { sousId, stakingToken, totalStaked, vaultKey } = pool
   const { totalCakeInVault } = useVaultPoolByKey(vaultKey)
+  const vaultPools = useVaultPools()
+  const cakeInVaults = Object.values(vaultPools).reduce((total, vault) => {
+    return total.plus(vault.totalCakeInVault)
+  }, BIG_ZERO)
```
Comment (chef-jojo) [inline src/views/Pools/components/PoolsTable/Cells/TotalStakedCell.tsx:28]: nice 👍

**Review by Chef-Cheems — APPROVED** [review]

**Pull request merged**